Assets/Prefabs/Player/Player.prefab

```diff
@@ -93,14 +93,13 @@ GameObject:
   m_Component:
```

Comment:
- Add `PlayerInteractionHandler` component
- Add "Gravity Gun" 

Idk how to handle GUI for player. For now only `PlayerInteractionHandler` requires UI, but in the future, there will be more components that connects to UI. I think, for now, we can create new prefab for in game UI and just manually drag'n'drop required references.